### ISSUE TITLE
Ticket Counter Real Estate Waste

### ISSUE THREAD

**Issue by simonchunk** [Repair-DeskPOS/RepairDesk-BUGS-IMPROVEMENTS#1544]
![aa1](https://user-images.githubusercontent.com/652173/111939440-61be2e00-8b20-11eb-821d-c9814798042c.jpeg)

**Comment by Uzair-RepairDesk** [comment]
Hi @simonchunk  Hope you are doing well. can you please share more information so that I can better understand your request?

Or if you want to limit the ticket status on your ticket counter then there is an option in the ticket counter setting.
![image](https://user-images.githubusercontent.com/22955861/111954163-7e9b3700-8b09-11eb-9d5b-f56643b6b75b.png)
For more details here is the link below.
https://help.repairdesk.co/knowledgebase/articles/1935100-ticket-counter-display

Hope this helps.
Thanks,

**Comment by simonchunk** [comment]
The entire top area is pointless. At a minimum, it should be much smaller in size. It takes up half the screen on PC and almost 2/3 on the iPad.

There is no use for see how many of each status we have as we can already see the status for each ticket

**Comment by Uzair-RepairDesk** [comment]
Hi @simonchunk  you can disable the ticket status from the ticket counter setting. but if you do, the ticket will also disappear with that specific status.

**Comment by simonchunk** [comment]
No, you've missed the point entirely. I want all the status available, but why would I want a list of the types of status at the top?

I already have the tickets listed, with the current status. What benefit are the giant numbers telling me how many I have for each status? My techs now need to scroll constantly because the numbers take up so much of the screen space

**Comment by Uzair-RepairDesk** [comment]
Got it, I have to share your request with the development team and check either it's possible or not, I will keep you posted or ask Danish to update you once I get the update from development team.

**Comment by simonchunk** [comment]
Fantastic. Here's a picture of the ipad for reference to see how inconvenient it is...

![1616399947974672316580475733407](https://user-images.githubusercontent.com/652173/111957908-b83b6480-8b40-11eb-8713-f930ac0be33a.jpg)
